### PR TITLE
Adds wall gazer abnormality. Lady facing the wall

### PR DESCRIPTION
**Pull request by EnderMage99**
## About The Pull Request

Added new abno - Wall gazer (Lady facing the wall) with ego gear and graphics. Missing sounds and new projectile graphics for the crossbow.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/teth/lady_facing_the_wall.dm [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	maxHealth = 400
+	health = 400
+	threat_level = TETH_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(55, 55, 55, 55, 55),
+		ABNORMALITY_WORK_INSIGHT = list(45, 45, 30, 30, 0),
+		ABNORMALITY_WORK_ATTACHMENT = list(100, 100, 100, 100, 100),
+		ABNORMALITY_WORK_REPRESSION = list(55, 55, 30, 30, 30)
+		)
+	pixel_x = -8
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/teth/lady_facing_the_wall.dm:15]: Set to -32

Comment (EnderMage99) [reply code/modules/mob/living/simple_animal/abnormality/teth/lady_facing_the_wall.dm]: All done, The Lady now touches the wall.

File: code/modules/projectiles/guns/ego_gun/he.dm [hunk 2]
```diff
@@ -187,3 +187,18 @@
...
+/obj/item/gun/ego_gun/wedge
+	name = "screaming wedge"
+	desc = "Hair has grown on the crossbow as if to express that the woman’s dejection will never be forgotten."
+	icon_state = "screamingwedge"
+	inhand_icon_state = "screamingwedge"
+	ammo_type = /obj/item/ammo_casing/caseless/ego_wedge
+	weapon_weight = WEAPON_HEAVY
+	fire_delay = 10
+	damtype = WHITE_DAMAGE
+	armortype = WHITE_DAMAGE
```
Comment (dsmith328) [inline code/modules/projectiles/guns/ego_gun/he.dm:200]: To my knowledge, this is not required and simply sets it's melee stats to this.

File: code/modules/mob/living/simple_animal/abnormality/teth/lady_facing_the_wall.dm [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+	return
+
+/mob/living/simple_animal/hostile/abnormality/wall_gazer/proc/scream()
+	//Add sound effects
+	for(var/mob/living/L in livinginrange(scream_range, src))
+		if(faction_check_mob(L, FALSE))
+			continue
+		if(L.stat == DEAD)
+			continue
+		L.apply_damage(scream_damage, WHITE_DAMAGE, null, L.run_armor_check(null, WHITE_DAMAGE), spread_damage = TRUE)
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/teth/lady_facing_the_wall.dm:51]: Add some sort of notification and I will merge after squashing

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Small changes

**Review by Kitsunemitsu — APPROVED** [review]
Approved! Welcome to the contributor team!

**Pull request merged**